plugins/BEdita/Core/src/Model/Table/ProfilesTable.php

```diff
@@ -105,5 +110,9 @@ public function beforeSave(Event $event, EntityInterface $entity)
         if (empty($entity->get('email'))) {
             $entity->set('email', null);
         }
+        if (empty($entity->get('title')) && (!empty($entity->get('name')) || !empty($entity->get('surname')))) {
```

Comment:
If possible I'd use `!$entity->has('title')` instead of `empty($entity->get('title'))`, and same with "name" and "surname". It's not exactly the same meaning, though, and it would behave differently with empty strings, but it would allow a forcibly empty title.

Reply:
done